drivers/serial/uart_stm32.c

```diff
@@ -1712,11 +1739,13 @@ STM32_UART_IRQ_HANDLER_DECL(index)					\
 									\
 PINCTRL_DT_INST_DEFINE(index);						\
 									\
+static const struct stm32_pclken pclken_##index[] =			\
+						   STM32_DT_INST_CLOCKS(index);\
```

Comment:
Please fix indentation:
```suggestion
					STM32_DT_INST_CLOCKS(index);\
```